install.sh

```diff
@@ -6,8 +6,11 @@ die() {
   exit 1
 }
 
+xdgBasherDir="${XDG_DATA_HOME:-$HOME/.local/share}/basher"
```

Comment:
Prefer `snake_case` rather than `camelCase` in Bash? I see other variables using snake_case already.

Reply:
Oops! I'll change that